.hlint.yaml

```diff
@@ -14,8 +14,10 @@
   - name: [ExistentialQuantification, MultiParamTypeClasses, NamedFieldPuns]
   - name: [FlexibleContexts, FlexibleInstances]
   - name: [PackageImports]
+  - {name: ConstraintKinds, within: [GHC.Util.Language.Haskell.GHC.ExactPrint.Types]}
```

Comment:
This is inoffensive, so just allow ConstraintKinds everywhere.

Reply:
✔️